requirements.txt

```diff
@@ -3,3 +3,5 @@ pydantic>=2.4,<2.5
 uvicorn>=0.24,<0.25
```

Comment:
Out of curiosity, is it worth updating the already listed dependencies (uvicorn is now at version 30 apparently)?

Reply:
I was tempted to do that @azangru but I couldn't make some of the BLAST features that are tied to uvicorn work and I made some adjustments to them , I'm not sure if a version update is gonna be a good idea. Andres should take a look at that later.